core/src/test/java/org/infinispan/expiration/impl/ClusterExpirationFunctionalTest.java

```diff
@@ -91,13 +91,14 @@ public void testLifespanExpiredOnBackupOwner() throws Exception {
    private void testLifespanExpiredEntryRetrieval(Cache<Object, String> primaryOwner, Cache<Object, String> backupOwner,
            ControlledTimeService timeService, boolean expireOnPrimary) throws Exception {
       MagicKey key = createKey(primaryOwner, backupOwner);
-      primaryOwner.put(key, key.toString(), 10, TimeUnit.MINUTES);
+      primaryOwner.put(key, key.toString(), 10, TimeUnit.MILLISECONDS);
 
       assertEquals(key.toString(), primaryOwner.get(key));
       assertEquals(key.toString(), backupOwner.get(key));
 
       // Now we expire on cache0, it should still exist on cache1
-      timeService.advance(TimeUnit.MINUTES.toMillis(10) + 1);
+      // Note this has to be within the buffer in RemoveExpiredCommand (100 ms the time of this commit)
```

Comment:
I don't understand this comment either, doesn't the entry have the same timestamp on both nodes?

Reply:
It has the same lifespan, but the created time can be off due to each node will insert at a slightly different time. For example:

n1: might insert at 1000
n2: might insert at 1013

due to network latency. This buffer is in case if we had a read from n1 at say 1004 and it tried to remove the entry from n2 at 1011, it would fail. This buffer is just to make it a little more lax.

Reply:
I agree in general, but this test uses `ControlledTimeService`, so the time on n2 should be the same as on n1

Reply:
There are two time services in this test and it only increments one of those.